lib/navigation/navigator.cpp

```diff
@@ -0,0 +1,44 @@
+#include "navigator.hpp"
+
+#include <cstdint>
+
+namespace hyped::navigation {
+
+Navigator::Navigator()
+{
+  // TODOLater: impement
+}
+
+void Navigator::navigate()
+{
+  /*
+  TODOLater: implement
+
+  main navigation control:
+
+  - look at new data
+  - preprocess
+  - check sensor agreement
+  - calculate current trajecotory
+  - update current trajectory
+  */
+}
+
+void Navigator::recordImuData(const core::RawImuData imu_data)
+{
+  for (std::size_t i = 0; i < core::kNumImus; ++i) {
+    raw_imu_data_.at(i) = imu_data.at(i);
+  }
```

Comment:
No, this is where data is taken into the navigation class to be passed around to everyone who needs it. Makes sense to wipe this for now tho since how to handle multiple data points at once is still tbd.